gen3-client/g3cmd/auth.go

```diff
@@ -0,0 +1,58 @@
+package g3cmd
+
+import (
+	"fmt"
+	"log"
+	"sort"
+
+	"github.com/spf13/cobra"
+	"github.com/uc-cdis/gen3-client/gen3-client/jwt"
+)
+
+func init() {
+
+	var authCmd = &cobra.Command{
+		Use:     "auth",
+		Short:   "Return data access priveleges from profile",
+		Long:    `Gets data access priveleges for specified profile.`,
```

Comment:
typo, should be `privileges`

Reply:
Fixed in [0878755](https://github.com/uc-cdis/cdis-data-client/pull/49/commits/08787553bc29e981c27d6efb6eef8ba50f39d9c7), thanks!

Reply:
there is another typo in the line above this line